docs/contributing/triaging-github-issues.md

```diff
@@ -3,7 +3,103 @@ title: Triaging GitHub Issues
+
+They typically:
+
+- answer questions that can be answered immediately
+  -test and reproduce possible bug reports and label them appropriately
+- communicate feature requests to the rest of the team and ensure a valid response
+- enable discussions on complicated use cases, whether themselves or via the rest of team
+
+## Why do we do first touch maintenance?
+
+We do first touch maintenance so that:
+
+- Questions are answered swiftly and correctly, therefore making users happy
+- Bug reports are reproducible and the most relevant data is collected before someone jumps in with a fix
+- Unrelated issues are resolved swiftly so we don't spend too much time on them
+
+## What is unique to Gatsby?
+
+Gatsby is unique among most open source projects because:
+
+- Gatsby integrates with many third party tools (WordPress, Drupal, Contentful etc) via source plugins and hence the typical scope of issues is widened significantly
+- Gatsby aims to be really beginner friendly (we want to be the new way someone gets started with web development) and this means that we need to accommodate a wide range of skill levels
+- At Gatsby, we've defined a couple of metrics that we measure to ensure we're responsive and helpful to our open source community
+
+## How do we do first touch maintenance?
+
+### General Guidelines
+
+- **Be empathic.** The author of an issue might be asking something that’s obvious to you but that doesn’t mean it’s obvious to them - it's important to consider the issue from the author’s viewpoint. People often remember how you make them feel, not what you told them.
+- **Add context.** When answering an issue, it can be useful to link to existing documentation, issues, PRs, or provide related context. This means the issue can serve as a reference to future readers.
+- **Encourage community contributions.** Getting people involved makes a huge impact. It’s often worth spending the time to write up a task as a `good first issue` instead of fixing the issue yourself. This can provide a low friction way for someone to get more involved in open source!
+- **Give issue authors time to close their own issues.** Sometimes, it might feel like an issue is resolved but the author could have follow up questions. It's usually best to give them a day or two to close the issue themselves.
+
+### Labelling
+
+Labelling helps group issues into manageable sets and also improves searchability and scannability. We have a set of labels that we use to group issues based on their type and status. While we want to limit adding too many labels, feel free to add one if it seems relevant and helps with this grouping!
+
+It's nice to update labels as the state of an issue changes or if for example, a question becomes a feature request or a bug report
+
+### Resolution Flow Chart
+
+https://whimsical.co/QvuMgo31T2C3xcWbou8xhy
+
+### Saved Replies
+
+https://github.com/orgs/gatsbyjs/teams/admin/discussions/3
```

Comment:
Is this publicly accessible?

Also probably worth documenting when to use these and why.

Reply:
If this is publicly accessible (it prompts me for a login, so probably not?), it would be good to have a sentence or two introducing the link.